suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -40,15 +40,23 @@ describe('Model Integration Test', () => {
   })
 
   test('Create a ModelInstanceDocument on one node and read it from another', async () => {
+    await waitForIndexingOrTimeout(
```

Comment:
can we move this into the test fixture setup?  ie at the end of the `beforeAll` block, right after we call `startIndexingModels`?  That way any tests added to this file later won't have to repeat this logic.  Also to keep more clear what is the actual behavior that the test case is focused on.